include/http/router.js

```diff
@@ -145,23 +146,39 @@ module.exports = function (pb) {
                     .use(this.router.allowedMethods());
 
                 this._addDefaultMiddleware();
-
-                this.__server = this.app.listen(port, () => {
-                    pb.log.info('PencilBlue is ready!');
-                });
+                const config = {
+                    https: {
+                        port: 8080,
+                        options: {
+                            key: fs.readFileSync('../OAuthFlows/key.pem', 'utf8'),
+                            cert: fs.readFileSync('../OAuthFlows/server.crt', 'utf8')
+                        }
+                    }
+                };
+                const serverCallback = this.app.callback();
+                const httpsServer = https.createServer(config.https.options, serverCallback);
+                this.__server = httpsServer
+                    .listen(config.https.port, function(err) {
+                        if (!!err) {
+                            pb.log.info('PencilBlue is not ready!');
```

Comment:
log the error here with error level logging not with info

Reply:
This would be treated in the card to enable https with KOA.

Reply:
Just pushed a new version with the card [TN-813](https://careerbuilder.atlassian.net/browse/TN-813) that has the protocol server configurable like in the old implementation.